bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -307,6 +315,13 @@ private void cleanup() {
         logger.debug("Doing cleanup");
         stopPolling();
         executions.clear();
+        if (httpClient != null) {
+            try {
+                httpClient.stop();
+            } catch (Exception e) {
```

Comment:
see above

Reply:
the same as above, it just throws a generic exception